libs/entry-components/configure-theming.md

```diff
@@ -269,9 +269,13 @@ $custom-theme: (
 ### 3. Fonts
 
 Allow customisation of typography and font styles based on Angular Material typography levels. `body` property is the most important and **mandatory** to set because that one is **default** and font family will be applied on all headers and buttons. If we want to additionally customize some other typography elements and properties, we can include other font properties.
-Each font related property in configuration can have **family** and **size** values.
+
+Each font related property in configuration can have: **family**, **size** and **letter-spacing** values.
 
 - `body`: Define typography for base body text.
+  - `family`: Set custom font family
```

Comment:
We're using set, specify and customize words to effectively describe the same action in one group of properties.
We also want to be consistent in regard whether we use dot in the end of sentence or not.

i.e. `body`: Defines typography for base body text.
  - `family`: Sets custom font family
  - `size`: Sets custom font size
  - `letter-spacing`:  Sets custom letter spacing;  if we leave default value or set it to `null` manually, Angular Material's default values will be applied